plugin/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -7,4 +7,23 @@ grant {
     permission java.lang.RuntimePermission "accessDeclaredMembers";
     permission java.lang.RuntimePermission "getClassLoader";
     permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
-};
+
+    // Upload model
+    permission java.net.SocketPermission "*", "connect,resolve";
```

Comment:
Do you think we can at least limit the port like: `*:443` or we also don't know about port?